zerver/tests/test_messages.py

```diff
@@ -2764,6 +2765,70 @@ def do_edit_message_assert_error(id_, unique_str, error):
         self.login(self.example_email("cordelia"))
         do_edit_message_assert_success(id_, 'D')
 
+    @mock.patch("zerver.lib.actions.send_event", return_value=None)
```

Comment:
Unless there's a specific reason to set `` return_value ``, I believe we generally just skip that and
`` @mock.patch("zerver.lib.actions.send_event") `` should be sufficient